internal/dataplane/parser/translators/httproute.go

```diff
@@ -1,30 +1,39 @@
 package translators
 
 import (
+	"encoding/json"
+	"fmt"
 	"sort"
-	"strconv"
 	"strings"
 
 	gatewayv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
 )
 
-// HTTPRouteTranslationMeta is a translation of a single HTTPRoute into metadata
+// KongServiceTranslation is a translation of a single HTTPRoute into metadata
 // that can be used to instantiate Kong routes and services.
-// Rules from this object should route traffic to BackendRefs from this object.
-type HTTPRouteTranslationMeta struct {
-	BackendRefs  []gatewayv1beta1.HTTPBackendRef
-	Rules        []gatewayv1beta1.HTTPRouteRule
-	RulesNumbers []int
+// Routes from this object should route traffic to BackendRefs from this object.
+type KongServiceTranslation struct {
```

Comment:
I'm hesitant about this name change. Are we expecting other than HTTP routes to get translators as well here? Why get rid of the HTTP from the name if the struct is specific for that protocol while the package it's in is not.

Reply:
The previous name was introduced by me in other PR. I prefer this one. It's totally internal package, we can change that name and it's usage in future.